src/coverlet.core/Logging/ILogger.cs

```diff
@@ -1,11 +1,17 @@
-namespace Coverlet.Console.Logging
+using System;
+
+namespace Coverlet.Core.Logging
 {
-    interface ILogger
+    public interface ILogger
     {
-        void LogSuccess(string message);
         void LogVerbose(string message);
+
```

Comment:
nit: remove extra line, keep formatting like original pls

Reply:
@derhally could you remove extraline to keep old formatting pls?

Reply:
@derhally can you revert this formatting(new lines)?I would see only green line on class visibility and new LogError method. We try to not change formatting of old files, it's to easy review and in case found bug/issue on updates.